diff --git a/scripts/autotools/libmongoc/Versions.m4 b/scripts/autotools/libmongoc/Versions.m4
index a6a5ddc..34ff754 100644
--- a/scripts/autotools/libmongoc/Versions.m4
+++ b/scripts/autotools/libmongoc/Versions.m4
@@ -1,14 +1,14 @@
 MONGOC_CURRENT_FILE=[]PHP_EXT_SRCDIR(mongodb)[/src/LIBMONGOC_VERSION_CURRENT]
-MONGOC_VERSION=$(cat $MONGOC_CURRENT_FILE)
+libmongoc_VERSION_FULL=$(cat $MONGOC_CURRENT_FILE)
 
 dnl Ensure newline for "cut" implementations that need it, e.g. HP-UX.
-MONGOC_MAJOR_VERSION=$( (cat $MONGOC_CURRENT_FILE; echo) | cut -d- -f1 | cut -d. -f1 )
-MONGOC_MINOR_VERSION=$( (cat $MONGOC_CURRENT_FILE; echo) | cut -d- -f1 | cut -d. -f2 )
-MONGOC_MICRO_VERSION=$( (cat $MONGOC_CURRENT_FILE; echo) | cut -d- -f1 | cut -d. -f3 )
-MONGOC_PRERELEASE_VERSION=$(cut -s -d- -f2 $MONGOC_CURRENT_FILE)
+libmongoc_VERSION_MAJOR=$( (cat $MONGOC_CURRENT_FILE; echo) | cut -d- -f1 | cut -d. -f1 )
+libmongoc_VERSION_MINOR=$( (cat $MONGOC_CURRENT_FILE; echo) | cut -d- -f1 | cut -d. -f2 )
+libmongoc_VERSION_PATCH=$( (cat $MONGOC_CURRENT_FILE; echo) | cut -d- -f1 | cut -d. -f3 )
+libmongoc_VERSION_PRERELEASE=$(cut -s -d- -f2 $MONGOC_CURRENT_FILE)
 
-AC_SUBST(MONGOC_VERSION)
-AC_SUBST(MONGOC_MAJOR_VERSION)
-AC_SUBST(MONGOC_MINOR_VERSION)
-AC_SUBST(MONGOC_MICRO_VERSION)
-AC_SUBST(MONGOC_PRERELEASE_VERSION)
+AC_SUBST(libmongoc_VERSION_FULL)
+AC_SUBST(libmongoc_VERSION_MAJOR)
+AC_SUBST(libmongoc_VERSION_MINOR)
+AC_SUBST(libmongoc_VERSION_PATCH)
+AC_SUBST(libmongoc_VERSION_PRERELEASE)
